innovation.game.php

```diff
@@ -8435,6 +8441,55 @@ function stPlayerInvolvedTurn() {
             case "182N1":
                 $step_max = 1;
                 break;
+
+            // id 196, Artifacts age 9: Luna 3
+            case "196N1":
+                $step_max = 1;
+                break;            
+
+            // id 199, Artifacts age 9: Philips Compact Cassette
+            case "199C1":
+                for($color = 0; $color < 5; $color++) {
+                    self::splay($player_id, $player_id, $color, 0); // Unsplay
+                }
+                break;
+
+            case "199N1":
```

Comment:
Okay, so I think "Splay up two colors on your board" can be implemented a lot more simply! First try implementing "Splay up any color on your board", and then repeat the effect a second time (by subtracting one from the step, like Oars).

"Splay up any color on your board" is very similar to the Flight card:

// id 77, age 8: Flight
        case "77N1A":
            // "You may splay any one color of your cards up"
            $options = array(
                'player_id' => $player_id,
                'n' => 1,
                'can_pass' => true,
                
                'splay_direction' => 3 /* up */
            );
            break;

Except in our case, we can't pass!

Reply:
done